pkg/actions/templates.go

```diff
@@ -24,12 +25,12 @@ import (
 // ListTemplates lists project templates of which Codewind is aware.
 // Filter them by providing flags
 func ListTemplates(c *cli.Context) {
-	templates, err := apiroutes.GetTemplates(
-		c.String("projectStyle"),
-		c.Bool("showEnabledOnly"),
-	)
-	if err != nil {
-		log.Printf("Error getting templates: %q", err)
+	projectStyle := c.String("projectStyle")
+	conID := strings.TrimSpace(strings.ToLower(c.String("conid")))
```

Comment:
You could pull `strings.TrimSpace(strings.ToLower(c.String("conid")))` out to a utility function since it is called so often.